### ISSUE TITLE
Seed Config within schema file

### ISSUE THREAD

**Issue by utamori** [prisma/prisma#15179]
## Problem

No place to set prisma seed command in environments without package.json (Deno, Rust, Dart, or others)

## Suggested solution

Allow seed settings to be written in prisma files

```prisma
seed development {
  command = "ts-node prisma/seed-dev.ts"
}

seed production {
  command = "ts-node prisma/seed.ts"
}
```

## Alternatives

Allow setting of the seed command by environment variables

## Additional context

- [prisma/prisma-client-go](https://github.com/prisma/prisma-client-go)
- [Brendonovich/prisma-client-rust](https://github.com/Brendonovich/prisma-client-rust)
- [odroe/prisma-dart](https://github.com/odroe/prisma-dart)

**Comment by jclaessens97** [comment]
Is it currently even possible to setup a different seed for production and for dev?

**Comment by janpio** [comment]
Yes, you can do whatever you want in your seed script. If your production and dev have differnet environment variables, just react to these in your script.

**Comment by jclaessens97** [comment]
Cool, thx
